documentation/asciidoc/computers/linux_kernel/configuring.adoc

```diff
@@ -1,49 +1,54 @@
+Configuration is most commonly done through the `make menuconfig` interface. Alternatively, you can modify your `.config` file manually, but this can be more difficult.
 
-=== Preparing to configure
+=== Prepare to configure
 
-The `menuconfig` tool requires the `ncurses` development headers to compile properly. These can be installed with the following command:
+The `menuconfig` tool requires the `ncurses` development headers to compile properly. To install these headers, run the following command:
 
-[,bash]
+[source,console]
 ----
- sudo apt install libncurses5-dev
+$ sudo apt install libncurses5-dev
 ----
 
-You'll also need to download and prepare your kernel sources, as described in the xref:linux_kernel.adoc#choosing_sources[build guide]. In particular, ensure you have installed the xref:linux_kernel.adoc#default_configuration[default configuration].
+Next, xref:linux_kernel.adoc#download-kernel-source[download your kernel sources]. In particular, ensure you have installed the xref:linux_kernel.adoc#apply-configuration[default configuration].
 
-=== Using `menuconfig`
+=== `menuconfig`
 
-Once you've got everything set up and ready to go, you can compile and run the `menuconfig` utility as follows:
+Once you've got everything set up, you can compile and run the `menuconfig` utility as follows:
 
-[,bash]
+[source,console]
 ----
- make menuconfig
+$ make menuconfig
 ----
 
-If you're cross-compiling a 32-bit kernel:
+To cross-compile a 64-bit kernel:
 
-[,bash]
+[source,console]
 ----
-make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- menuconfig
+$ make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- menuconfig
 ----
 
-Or, if you are cross-compiling a 64-bit kernel:
+To cross-compile a 32-bit kernel:
 
-[,bash]
+[source,console]
 ----
-make ARCH=arm64 CROSS_COMPILE=aarch64-linux-gnu- menuconfig
+$ make ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- menuconfig
 ----
 
-The `menuconfig` utility has simple keyboard navigation. After a brief compilation, you'll be presented with a list of submenus containing all the options you can configure. There are many options, so take your time to read through them and get acquainted.
+To navigate the `menuconfig` utility, use your keyboard:
 
-Use the arrow keys to navigate, the Enter key to enter a submenu (indicated by `+--->+`), Escape twice to go up a level or exit, and the space bar to cycle the state of an option. Some options have multiple choices, in which case they'll appear as a submenu and the Enter key will select an option. You can press `h` on most entries to get help about that specific option or menu.
+* to navigate directionally, use the *arrow keys*
+* to enter a submenu (indicated by `--->`), press the *Enter* key
+* to go up a level or exit, press *Escape* twice
+* to cycle the state of a binary option, press the *space bar*
```

Comment:
If it's definitely always a binary option, should this say "toggle" rather than "cycle" ?